<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBill</class>
 <widget class="QMainWindow" name="CreateBill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1284</width>
    <height>668</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1284</width>
    <height>668</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bill</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/img/logo/logo.png</normaloff>:/img/logo/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: white
;
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/logo/logo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 700 60pt &quot;Segoe UI&quot;;
color:black;</string>
        </property>
        <property name="text">
         <string>BillX</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 0, 0);
color: white;
font: 700 10pt &quot;Segoe UI&quot;;
border: 1pz solid black;
</string>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Bill Number :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_billno">
        <property name="styleSheet">
         <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Digital_clock">
        <property name="styleSheet">
         <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Clock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DateTime">
        <property name="styleSheet">
         <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Date And Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18,10,131);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;
</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18,10,131);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;</string>
        </property>
        <property name="text">
         <string>CLEAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18,10,131);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;
</string>
        </property>
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_print">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18,10,131);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;
</string>
        </property>
        <property name="text">
         <string>PRINT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Mobile Number :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_mobileno">
          <property name="styleSheet">
           <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Address : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_add">
          <property name="styleSheet">
           <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Product Code :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_productcode">
          <property name="styleSheet">
           <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Quantity : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_quantity">
          <property name="styleSheet">
           <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">border: 2px solid black</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Price :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_value">
      <property name="styleSheet">
       <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
border:1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
